.travis.yml

```diff
@@ -3,6 +3,7 @@ os:
   - osx
 
 language: rust
+sudo: true
```

Comment:
Travis has switched from containers to vm's since spectre/meltdown, so `sudo` doesn't do anything anymore.

Reply:
Yeah, I reverted the commit now.

It's also hard to understand the cause of the failure -- I contacted the Travis support, asking them to enable the debug mode for our repo.

For now, it seems the only option is to `#[ignore]` these tests temporarily. If there's no way to make it work with Travis, we could look into self-hosted GitHub Action runners. What do you think?

Reply:
I didn't know that travis had a debug mode.

Does Github Actions also prevent this ptrace usage?

Reply:
Yes, ptrace doesn’t work at all with GH Actions by default and seemingly there’s no way to elevate privileges (at least, I didn’t find any — this repo was setup with Linux GH Actions at first, and even process_vm_read didn’t work properly). There’s an option to use our own custom GH Actions runner which also allows to choose an operating system and install any software needed, so we might want to explore this in the future.